src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Declarations_SourceDeclarations.cs

```diff
@@ -56,5 +52,50 @@ public static Task<IEnumerable<ISymbol>> FindSourceDeclarationsAsync(Project pro
                 return declarations.SelectAsArray(t => t.Symbol);
             }
         }
+
+#pragma warning disable RS0026
```

Comment:
:bulb: Can you include the comment that says the title of this diagnostic?

Reply:
Sure